src/app/story/storyrow/totaltimes/totaltimes.component.css

```diff
@@ -0,0 +1,27 @@
+@media (min-width: 992px) {
+    .mobile-hide {
+        display: initial;
+    }
+}
+
+.mobile-hide {
```

Comment:
Optional: Instead of `mobile-hide` as a class name, I'd prefer a name describing what is being hidden such as `timeline-wrapper`.